backend/lib/adapters/gateways/dxtr/cases.dxtr.gateway.ts

```diff
@@ -294,7 +295,7 @@ export default class CasesDxtrGateway implements CasesInterface {
 
       divisionsAndCaseNumbers.forEach((caseNumbers, divisionCode) => {
         divisionAndCaseNumberParams.push(
-          `( cs.CS_DIV = '${divisionCode}' AND cs.CASE_ID IN ('${caseNumbers.join("', '")}')) `,
+          `( cs_div.CS_DIV_ACMS = '${divisionCode}' AND cs.CASE_ID IN ('${caseNumbers.join("', '")}')) `,
```

Comment:
I know this is not new, but our use of `divisionAndCaseNumberParams`, `parametersList`, and `params`, with type `string[]` in this gateway is a bit confusing given our typical use of `params` with a type of `DbTableFieldSpec[]`. I think I'd prefer something like `conditions` with modifiers here.